src/reporter.cr

```diff
@@ -8,5 +8,20 @@ module Spec2
     abstract def example_failed(example, exception)
     abstract def example_errored(example, exception)
     abstract def report
+
+    private def elapsed_time
+      elapsed_time = Time.now - Spec2.started_at
+      total_seconds = elapsed_time.total_seconds
+
+      if total_seconds < 1
+        output.puts "Finished in #{elapsed_time.total_milliseconds.round(2)} milliseconds"
```

Comment:
From the usage of this method, I derive that it should return just a string, not actually writing out to `output`.